sphinx/sphinx

```diff
@@ -0,0 +1 @@
+../sphinx
```

Comment:
What is this for (and also the `../examples` one above)?

Reply:
These are actually breaking our custom linter script at the moment, whatever they are

Reply:
```
           if path.startswith(exclude_dirs):
                continue
    
>           with io.open(path, 'r', encoding='utf-8') as file:
E           IsADirectoryError: [Errno 21] Is a directory: 'sphinx/examples'

```

Reply:
When I started working on Bokeh some time ago, without these symlinks, Sphinx was not able to find the example and some rst source files. That's why I had to add them.

Maybe the latest changes on `master` branch have fixed this and it's not needed anymore. I will remove them and trigger another build on Read the Docs.